Release.toml

```
@@ -74,3 +74,6 @@ version = "1.3.0"
     "migrate_v1.3.0_hostname-affects-etc-hosts.lz4",
     "migrate_v1.3.0_control-container-v0-5-2.lz4",
 ]
+"(1.3.0, 1.3.1)" = [
```

```suggestion
"(1.3.0, 1.4.0)" = [
```
(and associated changes)

It seems we've settled on minor version bumps unless it's a security release, which wouldn't include this change.